solr/core/src/java/org/apache/solr/handler/ClusterAPI.java

```diff
@@ -261,6 +263,13 @@ public void getNodes(SolrQueryRequest req, SolrQueryResponse rsp) {
     rsp.add("nodes", getCoreContainer().getZkController().getClusterState().getLiveNodes());
   }
 
+  @EndPoint(method = GET, path = "/cluster/cluster-status",  permission = COLL_READ_PERM)
```

Comment:
[+1] The method body itself here looks great 👍

[Q] My only question is about the path string you chose: "/cluster/cluster-status".

Did you pick this because "/cluster" was already taken by the `getCluster` method down below?  Or was there another reason?

Assuming the main factor was the conflict with `getCluster`, I'd vote to nuke that method and change the path string here to be "/cluster".

For reasons that I don't quite understand, the v2 API has two endpoints that expose the "List Collections" functionality: `ClusterAPI.getCluster` which we see below uses the "/cluster" path, and `CollectionsAPI.getCollections` [here](https://github.com/apache/solr/blob/c99af207c761ec34812ef1cc3054eb2804b7448b/solr/core/src/java/org/apache/solr/handler/CollectionsAPI.java#L79) which uses the path "/collections".  

There's absolutely no reason for that afaict, and "cluster status" is a very natural thing to expose at "/cluster".  So IMO we should be safe to delete `getCluster` in this PR.

Reply:
> Did you pick this because "/cluster" was already taken by the `getCluster` method down below? Or was there another reason?

Yes that's the main reason, "/cluster" had already been mapped to a method.

> For reasons that I don't quite understand, the v2 API has two endpoints that expose the "List Collections" functionality: `ClusterAPI.getCluster` which we see below uses the "/cluster" path, and `CollectionsAPI.getCollections` [here](https://github.com/apache/solr/blob/c99af207c761ec34812ef1cc3054eb2804b7448b/solr/core/src/java/org/apache/solr/handler/CollectionsAPI.java#L79) which uses the path "/collections".

Exactly. It was a challenge understing the same while working on the issue.

> There's absolutely no reason for that afaict, and "cluster status" is a very natural thing to expose at "/cluster". So IMO we should be safe to delete `getCluster` in this PR.

Will do that